docs/user-documentation.rst

```diff
@@ -0,0 +1,425 @@
+.. _user-documentation:
+
+
+Installation
+=============
+
+Here is a step by step guide on how to install Switchboard. It will help you
+download Switchboard and set it up with your application.
+
+Install Switchboard and its the dependencies using ``pip``::
+
+    pip install switchboard
+
+Next we need to bootstrap Switchboard within your application. The best approach
```

Comment:
Switching between 'you' and 'we' is confusing. I'd change this to 'you' for consistency.